### PR TITLE
Help and feedback changes

### PR DESCRIPTION
**Pull request by murrlipp**
### Description of the change

Updating 'Get help and contribute' partial to reflect new template design, and make changes across all building block pages.

### Release notes

- 'Suggest a change' section at the bottom of each page replaced with 'Get help and contribute' section.
- Removed from index and archive pages.
- Contributors included where relevant.
- Links to GitHub discussion for each building block added.

### CODE REVIEW

File: docs/_includes/example.njk [hunk 1]
```diff
@@ -70,7 +70,9 @@
 {% if figmaLink %}
   <div class="app-tabs__panel govuk-!-padding-2 app-tabs__panel--hidden" id="figma-default-{{ id }}" role="tabpanel" aria-labelledby="tab_figma-default-{{ id }}">
 
-To use the component in your design, add the MoJ Figma Kit to your Figma libraries and find the component using the Assets tab.
+This component is in the ‘Assets’ tab in the MoJ Figma Kit.
+
+If you’re external to MoJ, you can download the Kit and add it as a library to your Figma files. Updates to the Kit are only shown automatically for Figma users at MoJ, though, so you’ll need to add it again.
```
Comment (robertjmccarthy) [inline docs/_includes/example.njk:75]: I think the last part of this sentence reads a bit clunkily.

"_... though, so you'll need to add it again._"

File: docs/components/button-menu.md [hunk 2]
```diff
@@ -1,7 +1,9 @@
 ---
 layout: layouts/component-no-feedback.njk
```
Comment (robertjmccarthy) [inline docs/components/button-menu.md:2]: I think this should be using the component.njk layout instead of component-no-feedback.njk.

**Review by robertjmccarthy — COMMENTED** [review]
I've added 2 individual comments.

File: docs/components/button-menu.md (hunk 2 above)
Comment (murrlipp) [reply docs/components/button-menu.md:2]: Well spotted - thank you.  Will amend now.

File: docs/_includes/example.njk (hunk 1 above)
Comment (helennickols) [reply docs/_includes/example.njk:75]: OK. I'll look at this again.

**Comment by murrlipp** [comment]
I've just noticed a missing H3 'Get help' under 'Get help and contribute'. Added for the next push.

**Review by chrispymm — APPROVED** [review]

**Pull request merged**